src/enums.h

```diff
@@ -69,7 +73,7 @@ enum {
     ENTRY_SPECIAL,
     ENTRY_REVALIDATE_ALWAYS,
     DELAY_SENDING,
-    RELEASE_REQUEST,
+    RELEASE_REQUEST, ///< makes private key permanent
```

Comment:
Since the flag itself does not really make anything, I would rephrase as "prohibits making the key public".

Reply:
Done at  c0a470d.